client/src/i18n/en-US.json

```diff
@@ -349,11 +349,13 @@
     },
     "reviewer": {
       "title": "To Review",
-      "byline": "{type_name} in which you're assigned as a <strong>reviewer</strong>"
+      "byline": "{type_name} in which you're assigned as a <strong>reviewer</strong>",
+      "tooltip": "{type_name} with a status of Draft, Initially Submitted, or Awaiting Review will not appear here."
     },
     "coordinator": {
       "title": "To Coordinate",
-      "byline": "{type_name} in which you're assigned as a <strong>review coordinator</strong>"
+      "byline": "{type_name} in which you're assigned as a <strong>review coordinator</strong>",
+      "tooltip": "{type_name} with a status of Draft, Initially Submitted, or Awaiting Review will not appear here."
```

Comment:
You'll also want to add tooltips for the "dashboard" and "submissions" variations below in this file after lines 363, 367, 371, and 377. Otherwise, you'll get non-translated strings appearing in tooltips. 

I would have offered lines of code as official GitHub code suggestions but again, GitHub doesn't allow me to make suggestions on lines that haven't been modified. 

![Screenshot of the Submissions page showing the untranslated string submissions_tables.submissions.submitter.tooltip in a tooltip](https://user-images.githubusercontent.com/2668987/236511846-b359317b-44a5-4e2f-af7a-13ab6c192a36.png)